tests/integration/remote/invoke/test_remote_invoke.py

```diff
@@ -131,26 +202,38 @@ def setUpClass(cls):
         cls.stack_name = f"{TestMultipleResourcesInvoke.__name__}-{uuid.uuid4().hex}"
         cls.create_resources_and_boto_clients()
 
-    def test_invoke_empty_event_provided(self):
-        command_list = self.get_command_list(stack_name=self.stack_name, resource_id="EchoEventFunction")
+    @parameterized.expand([("EchoEventFunction", {})])
```

Comment:
We are going to add SFN into here once it is enabled, is that a right understanding?

Reply:
Initially I wanted to add that here, but the SFN resource used here requires passing in an event to run the Lambda functions. I can remove the `parameterized.expand` as it's not required here.